src/doubts/detail.html

```diff
@@ -281,8 +293,34 @@ <h2 class="text-sm font-medium text-gray-500">Topic</h2>
             <section aria-labelledby="activity-title" class="">
 
               <div>
-                <div class="pt-6">
+                <div class="mt-5" x-data="doubtForm()" x-show="hasQuery">
```

Comment:
Use x-cloak class with x-show everywhere. Otherwise the element is visible until alpinejs gets initialized.